### ISSUE TITLE
Geckoview NullPointerException

### ISSUE THREAD

**Issue by boek** [mozilla-mobile/focus-android#3618]
https://sentry.prod.mozaws.net/operations/focus-android/issues/4427405/

```
NullPointerException

NativeException NullHandle() [T = nsWindow::LayerViewSupport]

    org.mozilla.gecko.gfx.LayerSession$Compositor in setDefaultClearColor
    org.mozilla.gecko.gfx.CompositorController in onCompositorReady at line 35
    org.mozilla.gecko.gfx.LayerSession in onCompositorReady at line 444
    org.mozilla.gecko.gfx.LayerSession$1 in run at line 385
    android.os.Handler in handleCallback at line 789
    android.os.Handler in dispatchMessage at line 98
    android.os.Looper in loop at line 164
    android.app.ActivityThread in main at line 6944
    java.lang.reflect.Method in invoke
    com.android.internal.os.Zygote$MethodAndArgsCaller in run at line 327
    com.android.internal.os.ZygoteInit in main at line 1374
```

**Comment by ekager** [comment]
Filed in Bugzilla as [Bug 1499873](https://bugzilla.mozilla.org/show_bug.cgi?id=1499873)

**Comment by ekager** [comment]
According to GV team, this may have been fixed by [Bug 1492308](https://bugzilla.mozilla.org/show_bug.cgi?id=1492308), which is now in 63.

**Comment by boek** [comment]
I'm going to close this now, and it'll re-open if we find it when we audit crashers for a GV 63 build.
